client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -158,7 +158,7 @@ export const DateUtils = {
   isExpired: (expiryDate: Date | string | number): boolean =>
     isPast(expiryDate),
   isAlmostExpired: (
-    expiryDate: Date,
+    expiryDate: Date | string | number,
```

Comment:
Unrelated -- was initially going to use this, so updated input type to match the input of the function it calls. Didn't end up using it, but this is more correct so left it.